### PR TITLE
Preloader Added

### PR DESCRIPTION
**Pull request by danny007in**

### CODE REVIEW

**Comment by REJack** [comment]
works nice, I would only add a dark-mode override for it.

**Comment by danny007in** [comment]
> 
> 
> works nice, I would only add a dark-mode override for it.

You mean, Ur adding or I needed to add

**Comment by REJack** [comment]
I will add a small darkmode overwrite, but you should add a docs section for it.

File: index.html [hunk 1]
```diff
@@ -29,6 +29,11 @@
 <body class="hold-transition sidebar-mini layout-fixed">
 <div class="wrapper">
 
+  <!-- Preloader -->
+  <div class="preloader">
+    <img src="dist/img/AdminLTELogo.png" alt="AdminLTELogo" height="60">
```
Comment (XhmikosR) [inline index.html:34]: Always add width too

**Comment by REJack** [comment]
@danny007in Add a docs section for the preloader then I will merge it, so we can publish the v3.1.0.

**Comment by danny007in** [comment]
Is dark mode preloader added or not

**Comment by REJack** [comment]
No I have no write access to your PR, I will add it after merge.

**Comment by danny007in** [comment]
u can try now, i have allow from earlier

**Comment by REJack** [comment]
I've added the darkmode, now you can add the docs section 😄

**Comment by danny007in** [comment]
ok i will check doc properly and update it

**Comment by danny007in** [comment]
and u needed to remove unwanted construct in TODO at dropdown.js

**Comment by REJack** [comment]
> and u needed to remove unwanted construct in TODO at dropdown.js

ignore it

**Comment by buncis** [comment]
did it mean to fix this bug? https://github.com/ColorlibHQ/AdminLTE/discussions/3319

and also if that is true how about adding preloader to starter page too ?

**Pull request merged**